src/components/TodoItem.js

```diff
@@ -1,75 +1,83 @@
-import React, { useEffect, useState } from "react"
-import styles from "./TodoItem.module.css"
-import { FaTrash } from "react-icons/fa"
+import React, { useState } from 'react';
+import { FaTrash } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+import styles from './TodoItem.module.css';
 
 const TodoItem = (props) => {
+  const [isEditing, setEditing] = useState(false);
 
-    const [isEditing, setEditing] = useState(false);
+  const completedStyle = {
+    fontStyle: 'italic',
+    color: '#595959',
+    opacity: 0.4,
+    textDecoration: 'line-through',
+  };
 
-    const completedStyle = {
-        fontStyle: "italic",
-        color: "#595959",
-        opacity: 0.4,
-        textDecoration: "line-through",
-    }
-
-    const { completed, id, title } = props.todo
-    const { handleChangeProps, delTodo, setUpdate } = props
-    const { item, textInput } = styles
+  // Destructuring props
+  const { todo } = props;
+  const { completed, id, title } = todo;
+  const { handleChange, delTodo, setUpdate } = props;
+  const { item, textInput } = styles;
 
-    const handleEditing = () => {
-        setEditing(true);
-    }
+  const handleEditing = () => {
+    setEditing(true);
+  };
 
-    let viewMode = {}
-    let editMode = {}
+  const viewMode = {};
+  const editMode = {};
 
-    if (isEditing) {
-        viewMode.display = "none"
-    } else {
-        editMode.display = "none"
-    }
+  if (isEditing) {
+    viewMode.display = 'none';
+  } else {
+    editMode.display = 'none';
+  }
 
-    const handleUpdatedDone = event => {
-        if (event.key === "Enter") {
-            setEditing(false)
-        }
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
     }
+  };
 
-    useEffect(() => {
-        return () => {
-            console.log("Cleaning up...")
```

Comment:
Please consider removing `console.log` as this does not conform to the [Javascript best practises ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

Reply:
Thanks for the comment @JonahKayizzi but this was addressed already, I believe you didn't review the right version